linux_p2/.gitignore

```diff
@@ -0,0 +1,4 @@
+### Linux ###
+*.txt
+*.log
```

Comment:
its better to have only one .gitignore, in the root repo, and there set:
*.txt
*.log
*.tar.gz
etc